server/routes/overview/postInductionOverviewView.ts

```diff
@@ -1,6 +1,6 @@
 import type { ActionPlan, FunctionalSkills, InPrisonEducationRecords, PrisonerSummary } from 'viewModels'
 
-export default class OverviewView {
+export default class PostInductionOverviewView {
```

Comment:
Renamed this view class from `OverviewView` to `PostInductionOverviewView` - it is only used post-induction because it contains an `ActionPlan`